types/mailgun-js/index.d.ts

```diff
@@ -3,7 +3,7 @@
 // Definitions by: Sampson Oliver <https://github.com/sampsonjoliver>
 //                 Andi Pätzold <https://github.com/andipaetzold>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.2
```

Comment:
What's the reason for the version change? Which type definition requires TypeScript 2.3?

Reply:
When running the linter it says that this changeset fails for 2.2 but passes on 2.3. I'm not actually sure which specific feature requires 2.3, but it is reasonably old that I don't think it makes a big difference?

Reply:
Strange - I just tested it and linting worked for both versions 🤔 
But as you said, it's a quite old version (Feb 2017 vs April 2017).